### PR TITLE
Extend List of some good tutorials

### LINKED ISSUE #18: Add Some Good Tutorial
You can add tutorial where anyone can start there journey to become blockchian developer with practical implementation.

### PR DESCRIPTION
**Pull request by smit-1923**
Fixes #18

I added some amazing development videos for blockchain. So that any one who want to implement in practically they can benefited through this.

### CODE REVIEW

**Comment by shuklaritvik06** [comment]
Hey @smit-1923 your PR looks good but please create a issue first and then link that with this PR :)

**Review by shuklaritvik06 — APPROVED** [review]
Else looks good :)

File: learning-guide/beginner.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Click the image to view on YouTube.
 | --- | :-- | --- |
 | Full Roadmap to learn Blockchain development in 2021 | [![video](https://img.youtube.com/vi/ci_AIMCF-HA/0.jpg)](https://www.youtube.com/watch?v=ci_AIMCF-HA) | 12m 39s |
 | How to become a blockchain developer in 2021 | [![video](https://img.youtube.com/vi/OwSl2xwl2-w/0.jpg)](https://www.youtube.com/watch?v=OwSl2xwl2-w) | 1h 28m 20s |
+| How To Become a Blockchain Developer (Full Roadmap) | [<img src="https://user-images.githubusercontent.com/68073132/135997177-23e759e3-2e18-4196-b1d7-746094edb237.jpeg" height="250" width="500">](https://www.youtube.com/watch?v=P87pLayUD8c) | 14m 13s |
+| Complete Blockchain Development Roadmap by Anuj Bhaiya | [<img src="https://user-images.githubusercontent.com/68073132/135997361-a5f6f10a-5f7d-42ce-bd9c-265484b84deb.jpeg" height="250" width="500">](https://www.youtube.com/watch?v=n_5oeZWbseY)| 17m 51s |
```
[the remark this answers is not in the document]
Comment (vinzvinci) [reply learning-guide/beginner.md:10]: Yes please, we are a global community. We prefer English version only

File: learning-guide/beginner.md [hunk 2]
```diff
@@ -15,6 +18,8 @@ Click the image to view on YouTube.
 | --- | :-- | --- |
 | The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
 | Local Solidity Development | [View](https://hardhat.org/) |
+| Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial | [![video](https://user-images.githubusercontent.com/68073132/135998915-31e05399-0495-493a-920a-1a3ddd939cf2.jpeg)](https://youtu.be/M576WGiDBdQ) | 16h 22m 12s
+| How to Build a Full Stack NFT Marketplace on Ethereum with Polygon and Next.js | [![video](https://user-images.githubusercontent.com/68073132/136000852-6c23261c-d1f1-4c0f-ba53-c406c6b6ce14.jpeg)](https://youtu.be/GKJBEEXUha0) | 2h 21m 22s
```
Comment (vinzvinci) [inline learning-guide/beginner.md:22]: > It is best to separate the written content, just like the below:

## Development Guide

Click the image to view on YouTube.

| Title | Content | Length | 
| --- | :-- | --- |
| The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [video](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
| Solidity, Blockchain, and Smart Contract Course – Beginner to Expert Python Tutorial | [video](https://youtu.be/M576WGiDBdQ) | 16h 22m 12s
| How to Build a Full Stack NFT Marketplace on Ethereum with Polygon and Next.js | [video](https://youtu.be/GKJBEEXUha0) | 2h 21m 22s

## Written Content
| Title | Content |
| --- | :-- |
| Local Solidity Development | [View](https://hardhat.org/) |

File: learning-guide/beginner.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (smit-1923) [reply learning-guide/beginner.md]: Sorry, I thought that all videos are in english. Now, You can check, I deleted that one. If any more conflicts are there then do let me know. I will be changed asap.

File: learning-guide/beginner.md (hunk 2 above)
Comment (vinzvinci) [reply learning-guide/beginner.md:21]: How about this one?

Comment (smit-1923) [reply learning-guide/beginner.md:21]: Okay, I will change it right now.

File: learning-guide/beginner.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Click the image to view on YouTube.
 | Title | Content | Length | 
 | --- | :-- | --- |
 | The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
-| Local Solidity Development | [View](https://hardhat.org/) |
+| Local Solidity Development | [![video](https://user-images.githubusercontent.com/68073132/136015000-4801b909-f183-4726-b2c5-e2ce82aa2d31.png)](https://hardhat.org/) |
```
Comment (vinzvinci) [inline learning-guide/beginner.md:19]: Could you write in length = Written content? Sample below

| Title | Content | Length | 
| --- | :-- | --- |
| Local Solidity Development | [video](https://hardhat.org/) | Written Content |

Comment (smit-1923) [reply learning-guide/beginner.md:19]: Sure

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**